lighthouse-core/lib/page-functions.js

```diff
@@ -107,12 +107,20 @@ function getElementsInDocument(selector) {
 /**
  * Gets the opening tag text of the given node.
  * @param {Element} element
+ * @param {Array<string>} ignore
```

Comment:
`@param {Array<string>}= ignoreAttrs` (`=` to mark it as optional)

Reply:
`@param {Array<string>=}` or `@param {Array<string>}=`  ?

Reply:
whoops, yes, inside `{Array<string>=} `